libs/angular/account/src/lib/components/enrollments/account-enrollments.view.component.ts

```diff
@@ -0,0 +1,67 @@
+import { NgForOf, NgIf, NgTemplateOutlet } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+import { SemesterEnrollment } from '@sol/classes/domain';
+import {
+    Requested,
+    SolLoadedDirective,
+    SolLoadingDirective,
+    SolErrorDirective,
+    SolEmptyDirective,
+} from '@sol/angular/request';
+import { EnrollmentViewComponent } from './enrollment.view.component';
+import { EnrollmentSkeletonViewComponent } from './enrollment-skeleton.view.component';
+
+@Component({
+    selector: 'sol-account-enrollments-view',
+    template: `<h2>Enrollments</h2>
+        <ng-container *solEmpty="enrollments">
+            <ng-container *ngTemplateOutlet="skeleton"></ng-container>
+        </ng-container>
+        <ng-container *solLoading="enrollments">
+            <ng-container *ngTemplateOutlet="skeleton"></ng-container>
+        </ng-container>
+        <ng-container *solError="enrollments">
+            <p>There was an error loading your enrollments.</p>
+        </ng-container>
+        <ng-container *solLoaded="enrollments">
+            <ng-container *ngIf="enrollments.length > 0; else empty">
```

Comment:
Migrated to new slot-based view templating for requests.

Now requests are displayed using `solEmpty`, `solLoading`, `solError`, and `solLoaded` slots.